modules/oadp-ssec-encrypted-backups.adoc

```diff
@@ -0,0 +1,154 @@
+
+* To make {oadp-short} mount a secret to the Velero pod at `/credentials`, use the following default secret name for AWS: `cloud-credentials` and leave `dpa.spec.backupLocations[].velero.credential` or `dpa.spec.snapshotLocations[].velero.credential` empty. This is a workaround for a known issue that will be solved in the future as part of https://github.com/vmware-tanzu/velero/issues/7767 .
+
+[NOTE]
+====
+In the procedure that follows, there is an example of `backupLocations` which does not specify credentials. This example would trigger an OADP secret mounting.
+====
+
+* If you need `backupLocations` to have credentials with a different name than `cloud-credentials`, you must add a `snapshotLocation` like below which does not contain a credential name, meaning the `snapshotLocation` will use `cloud-credentials` as its secret for taking snapshots.
+
+.Example `snapshotLocation` in a DPA without credentials specified.
+
+[source,yaml]
+----
+ snapshotLocations:
+  - velero:
+      config:
+        profile: default
+        region: <region>
+      provider: aws
+# ...
+----
+.Procedure
+. Create an SSE-C encryption key by running the following command:
++
+[source,terminal]
+----
+$ dd if=/dev/urandom bs=1 count=32 > sse.key
+cat sse.key | base64 > sse_encoded.key
+ln -s sse_encoded.key customer-key
+----
+. Create an OpenShift secret.
+.. If you are initially installing and configuring {oadp-short}, you can create the AWS credential and encryption key secret at the same time by running the following command:
++
+[source,terminal]
+----
+$ oc create secret generic cloud-credentials --namespace openshift-adp --from-file cloud=<path>/openshift_aws_credentials,customer-key=<path>/sse_encoded.key
+----
+.. If you are updating an existing installation, edit the values of the `cloud-credential` `secret` block of the `DataProtectionApplication` CR manifest, as in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  cloud: W2Rfa2V5X2lkPSJBS0lBVkJRWUIyRkQ0TlFHRFFPQiIKYXdzX3NlY3JldF9hY2Nlc3Nfa2V5P<snip>rUE1mNWVSbTN5K2FpeWhUTUQyQk1WZHBOIgo=
+  customer-key: v+<snip>TFIiq6aaXPbj8dhos=
+kind: Secret
+# ...
+----
+. Edit the value of the `customerKeyEncryptionFile` attribute in the `backupLocations` block of the `DataProtectionApplication` CR manifest, as in the following example:
++
+[source,yaml]
+----
+spec:
+  backupLocations:
+    - velero:
+        config:
+          customerKeyEncryptionFile: /credentials/customer-key
+          profile: default
+# ...
+----
++
+[WARNING]
+====
+Existing installations require you to restart the Velero pod to remount the secret credentials properly.
+====
++
+The installation is complete, and you can back up and restore OpenShift resources. The data saved in AWS S3 storage is encrypted with the new key, and you cannot download it from the AWS S3 console or API without the additional encryption key.
```

Comment:
If actually wonder if we should refer to "OpenShift" rather than "OpenShift Container Platform" more often than not, but since the current document uses the latter pretty consistently, I made the change you suggested. The bigger question is for another day (and I might be dead wrong, too).